### PR TITLE
KevS-SK1780-Update-Retriving-logs-for-WebCloudDatabases

### PR DESCRIPTION
**Pull request by Kazarus4**
FRFR doing

### CODE REVIEW

**Comment by Kazarus4** [comment]
All translations are done.

Need proofs for all subs.

**Review by bs-ovh — APPROVED** [review]

File: pages/web_cloud/web_cloud_databases/retrieve-logs/guide.it-it.md [hunk 1]
```diff
@@ -1,48 +1,61 @@
...
+- Informazioni aggiuntive sull'evento (pagina o file consultato, applicazione avviata, server remoto chiamato, nome di un file caricato o scaricato, ecc.)
+- l'origine dell'evento (identificativo dell'utente, indirizzo IP sorgente, programma sorgente, ecc.).
+- lo stato del sistema in cui si svolge l'evento (risorse disponibili, memoria residua, utilizzo della CPU, ecc.).
+
+Nella maggior parte dei casi, i log vengono generati direttamente dai sistemi informatici in cui si verificano gli eventi.
+e vengono memorizzati e memorizzati in file di testo, detti anche file di log.
+
+In questo modo, i file di log permettono di effettuare le seguenti azioni:
+
+- analizzare il comportamento del sistema informatico che genera i log.
```
Comment (Y0Coss) [inline pages/web_cloud/web_cloud_databases/retrieve-logs/guide.it-it.md]:
```suggestion
- Analizzare il comportamento del sistema informatico che genera i log.
```

File: pages/web_cloud/web_cloud_databases/retrieve-logs/guide.it-it.md [hunk 2]
```diff
@@ -1,48 +1,61 @@
...
->
-> **Questa guida è attualmente in corso di aggiornamento. Alcune informazioni potrebbero essere incomplete o obsolete. Grazie per la comprensione.**
-
 ## Obiettivo
 
-La soluzione [Web Cloud Databases](/links/web/databases) permette di ospitare più database. In alcune situazioni, potrebbe essere necessario consultare / recuperare i log:
+Un log corrisponde a un evento che si è verificato su un sistema informatico (server, computer, applicazione, sito Web, database, rete informatica, ecc.).
+Ad esempio, un log può salvare e contenere uno o più dei seguenti elementi:
+
+- la data e l'ora (data, ora, minuto, secondo, ecc.) dell'evento.
```
Comment (Y0Coss) [inline pages/web_cloud/web_cloud_databases/retrieve-logs/guide.it-it.md]:
```suggestion
- La data e l'ora (data, ora, minuto, secondo, ecc.) dell'evento.
```

File: pages/web_cloud/web_cloud_databases/retrieve-logs/guide.it-it.md [hunk 3]
```diff
@@ -1,48 +1,61 @@
...
+- la natura dell'evento (connessione, disconnessione, errore, download, upload, alert, ecc.).
```
Comment (Y0Coss) [inline pages/web_cloud/web_cloud_databases/retrieve-logs/guide.it-it.md]:
```suggestion
- La natura dell'evento (connessione, disconnessione, errore, download, upload, alert, ecc.).
```

File: pages/web_cloud/web_cloud_databases/retrieve-logs/guide.it-it.md [hunk 4]
```diff
@@ -1,48 +1,61 @@
...
+- Informazioni aggiuntive sull'evento (pagina o file consultato, applicazione avviata, server remoto chiamato, nome di un file caricato o scaricato, ecc.)
+- l'origine dell'evento (identificativo dell'utente, indirizzo IP sorgente, programma sorgente, ecc.).
```
Comment (Y0Coss) [inline pages/web_cloud/web_cloud_databases/retrieve-logs/guide.it-it.md]:
```suggestion
- L'origine dell'evento (identificativo dell'utente, indirizzo IP sorgente, programma sorgente, ecc.).
```

File: pages/web_cloud/web_cloud_databases/retrieve-logs/guide.it-it.md [hunk 5]
```diff
@@ -1,48 +1,61 @@
...
+- lo stato del sistema in cui si svolge l'evento (risorse disponibili, memoria residua, utilizzo della CPU, ecc.).
```
Comment (Y0Coss) [inline pages/web_cloud/web_cloud_databases/retrieve-logs/guide.it-it.md]:
```suggestion
- Lo stato del sistema in cui si svolge l'evento (risorse disponibili, memoria residua, utilizzo della CPU, ecc.).
```

File: pages/web_cloud/web_cloud_databases/retrieve-logs/guide.pt-pt.md [hunk 6]
```diff
@@ -1,48 +1,61 @@
...
 
-**Saiba como recuperar os logs das suas bases de dados alojadas no servidor Web Cloud Databases**
+**Descubra como visualizar e gerir os logs do seu serviço Web Cloud Databases.**
 
 ## Requisitos
 
-- Ter uma solução [Web Cloud Databases](/links/web/databases) (incluída ou não numa oferta de [alojamento web performance](/links/web/hosting)).
-Ter acesso à [Área de Cliente OVHcloud](/links/manager){.external}.
+- Ter uma solução [Web Cloud Databases](/links/web/databases).
+Ter acesso à [Área de Cliente OVHcloud](/links/manager).
```
Comment (Y0Coss) [inline pages/web_cloud/web_cloud_databases/retrieve-logs/guide.pt-pt.md]:
```suggestion
- Ter acesso à [Área de Cliente OVHcloud](/links/manager).
```

File: pages/web_cloud/web_cloud_databases/retrieve-logs/guide.pt-pt.md [hunk 7]
```diff
@@ -101,9 +114,66 @@ Poderá descarregá-lo para o seu computador para o consultar.
...
+3. Na coluna da esquerda, clique no menu pendente `Web Cloud Databases`{.action}.
+4. Selecione a instância Web Cloud Databases em causa.
+5. Na página que vai aparecer, clique no separador `Logs (registos)`{.action}.
+6. À direita da caixa onde são apresentados os logs em tempo real, clique no botão `Subscrever`{.action}.
+
+Na nova página que se abrir, e se dispuser de várias soluções Logs Data Platform na sua [Área de Cliente OVHcloud](/links/manager), selecione, na lista pendente situada imediatamente abaixo do botão intitulado `Adicionar um fluxo de dados`, a referência do Logs Data Platform com a qual deseja subscrever.
+
+Ocorrem dois casos para subscrever a solução Web Cloud Databases.
+
+#### Cas n°1 - Subscrever a um fluxo já existente na sua solução Logs Data Platform <a name="wcdb-ldp-case1"></a>
```
Comment (Y0Coss) [inline pages/web_cloud/web_cloud_databases/retrieve-logs/guide.pt-pt.md]: "Cas" in PT ?

Comment (Kazarus4) [inline pages/web_cloud/web_cloud_databases/retrieve-logs/guide.pt-pt.md]:
```suggestion
#### Caso n°1 - Subscrever a um fluxo já existente na sua solução Logs Data Platform <a name="wcdb-ldp-case1"></a>
```

File: pages/web_cloud/web_cloud_databases/retrieve-logs/guide.pt-pt.md [hunk 8]
```diff
@@ -101,9 +114,66 @@ Poderá descarregá-lo para o seu computador para o consultar.
...
+
+Se o fluxo em causa já existir, este aparecerá sob a forma de uma linha na tabela situada na parte inferior da página.
+Neste caso preciso, e para subscrever a sua solução Web Cloud Databases a este fluxo existente, basta clicar no botão `Subscrever`{.action} à direita da linha correspondente ao fluxo em causa.
+
+Após alguns segundos e se permanecer na mesma página, será apresentada uma mensagem na Área de Cliente a indicar que a subscrição foi criada com sucesso.
+
+#### Cas n°2 - Subscrever um novo fluxo de dados na sua solução Logs Data Platform
```
Comment (Kazarus4) [inline pages/web_cloud/web_cloud_databases/retrieve-logs/guide.pt-pt.md]:
```suggestion
#### Caso n°2 - Subscrever um novo fluxo de dados na sua solução Logs Data Platform
```

**Pull request merged**